codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -310,14 +310,9 @@ class ServerProtocolTestGenerator(
         checkForbidHeaders(this, "&http_response.headers()", testCase.forbidHeaders)
         checkRequiredHeaders(this, "&http_response.headers()", testCase.requireHeaders)
         checkHttpResponseExtensions(this)
-        if (!testCase.body.isEmpty) {
-            rustTemplate(
-                """
-                let body = #{Hyper}::body::to_bytes(http_response.into_body()).await.expect("unable to extract body to bytes");
-                #{AssertEq}(${escape(testCase.body.get()).dq()}, body);
-                """,
-                *codegenScope
-            )
+        // "If no request body is defined, then no assertions are made about the body of the message."
```

Comment:
```suggestion
        // If no request body is defined, then no assertions are made about the body of the message.
```

Reply:
This is intentionally in quotes because I'm citing the Smithy spec.

Reply:
Don't worry, it just felt weird to me..